RecoTracker/IterativeTracking/python/DetachedTripletStep_cff.py

```diff
@@ -252,15 +254,20 @@
 detachedTripletStep = ClassifierMerger.clone()
 detachedTripletStep.inputClassifiers=['detachedTripletStepClassifier1','detachedTripletStepClassifier2']
 
+trackingPhase1.toReplaceWith(detachedTripletStep, detachedTripletStepClassifier1.clone(
+     mva = dict(GBRForestLabel = 'MVASelectorDetachedTripletStep_Phase1'),
+     src = 'detachedTripletStepTracks',
+     qualityCuts = [-0.2,0.3,0.8]
+))
+(trackingPhase1 & fastSim).toModify(detachedTripletStep,vertices = "firstStepPrimaryVerticesBeforeMixing")
 
-#LWTNN selector
 from RecoTracker.FinalTrackSelectors.TrackLwtnnClassifier_cfi import *
 from RecoTracker.FinalTrackSelectors.trackSelectionLwtnn_cfi import *
-trackingPhase1.toReplaceWith(detachedTripletStep, TrackLwtnnClassifier.clone(
+trackdnn.toReplaceWith(detachedTripletStep, TrackLwtnnClassifier.clone(
      src = 'detachedTripletStepTracks',
      qualityCuts = [0.0, 0.4, 0.8],
 ))
-(trackingPhase1 & fastSim).toModify(detachedTripletStep,vertices = "firstStepPrimaryVerticesBeforeMixing")
+(trackdnn & fastSim).toModify(detachedTripletStep,vertices = "firstStepPrimaryVerticesBeforeMixing")
 
 highBetaStar_2018.toReplaceWith(detachedTripletStep, detachedTripletStepClassifier1.clone(
      qualityCuts = [-0.5,0.0,0.5],
```

Comment:
this and pp_on_AA below were previously simpler, using toModify

Reply:
Changed to simpler versions using toModify